stdlib/public/SwiftShims/LibcOverlayShims.h

```diff
@@ -116,4 +116,19 @@ int static inline _swift_stdlib_openat(int fd, const char *path, int oflag,
 #pragma clang assume_nonnull end
 #endif
 
+#if defined(__OpenBSD__)
+#include <stdio.h>
+static inline FILE *_swift_stdlib_stdin(void) {
```

Comment:
These should be enclosed in nullability pragmas if possible. Right now, they'll import as IUOs.

Reply:
Bumped this up to enclose in the assume_nonnull pragma.